tests/Unit/NumericalAlgorithms/Spectral/Test_Spectral.cpp

```diff
@@ -446,6 +446,58 @@ void test_spectral_quantities_for_mesh() {
                                          Spectral::Quadrature::GaussLobatto>(
       mesh2d.slice_through(1));
 }
+
+void test_boundary_lagrange_polynomials() noexcept {
```

Comment:
[nitpick] it's not super clear from this name that you're testing interpolation onto the boundary and lifting from the boundary. I can't think of an improved name that isn't as long as a that entire sentence... but perhaps adding a quick code comment to clarify what is being tested would be useful here. Perhaps something like,
```
// Test interpolation from volume Gauss points to the boundary, and lifting
// terms from the boundary back to the volume Gauss points
```